FirebaseAI/Sources/Types/Public/Backend.swift

```diff
@@ -25,26 +25,30 @@ public struct Backend {
   ///     for a list of supported locations.
   public static func vertexAI(location: String = "us-central1") -> Backend {
     return Backend(
-      apiConfig: APIConfig(service: .vertexAI(endpoint: .firebaseProxyProd), version: .v1beta),
-      location: location
+      apiConfig: APIConfig(
+        service: .vertexAI(endpoint: .firebaseProxyProd),
+        version: .v1beta,
+        location: location
```

Comment:
To illustrate above suggestion:
```suggestion
        service: .vertexAI(endpoint: .firebaseProxyProd, location: location),
        version: .v1beta,
```